src/components/Savings/Stats/Stats.tsx

```diff
@@ -31,9 +29,50 @@ const Stats = () => {
     });
   }, [currentSavings]);
 
+  const currentYieldFormatted = useMemo(() => {
+    if (!currentYield) {
+      return null;
+    }
+
+    return DecimalFormat.format(currentYield, {
+      style: 'percentage',
+      fractionDigits: R_TOKEN_UI_PRECISION,
```

Comment:
we should use `R_TOKEN_UI_PRECISION` when formating percentage. Hardcode it to 2, or create new contant for percentage precision.